jmx/src/main/java/org/jboss/as/jmx/PluggableMBeanServerImpl.java

```diff
@@ -1296,6 +1298,8 @@ static void doLog(String userId, ManagedAuditLogger auditLogger, boolean readOnl
 
         private final MBeanServer delegate;
 
+        private HashMap<NotificationListener, NotificationListener> listeners = new HashMap<>();
```

Comment:
This field should be final.  

Also, as a matter of code style please make the type of the field Map instead of HashMap.

Reply:
@bstansberry , I am done with provided review, Please verify.
Thanks.